corehq/apps/data_interfaces/views.py

```diff
@@ -155,6 +155,69 @@ def get_deleted_item_data(self, item_id):
         }
 
 
+class ArchiveFormView(DataInterfaceSection):
```

Comment:
ah that's definitely not superuser, that's just a domain permission. if you want to make this superuser only you have to add a check for `is_superuser` somewhere.